l1-contracts/src/core/Rollup.sol

```diff
@@ -555,13 +577,13 @@ contract Rollup is EIP712("Aztec Rollup", "1"), Leonidas, IRollup, ITestRollup {
    * @param  _epochSize - The size of the epoch (to be promoted to a constant)
    * @param  _args - Array of public inputs to the proof (previousArchive, endArchive, previousBlockHash, endBlockHash, endTimestamp, outHash, proverId)
    * @param  _fees - Array of recipient-value pairs with fees to be distributed for the epoch
-   * @param  _aggregationObject - The aggregation object for the proof
+   * @param  _blobPublicInputsAndAggregationObject - The aggregation object and blob PIs for the proof
    */
   function getEpochProofPublicInputs(
     uint256 _epochSize,
     bytes32[7] calldata _args,
     bytes32[] calldata _fees,
-    bytes calldata _aggregationObject
+    bytes calldata _blobPublicInputsAndAggregationObject // having separate inputs here caused stack too deep
```

Comment:
As earlier, using a struct can be used to work around this while keeping the readability.